progressbar_printer.go

```diff
@@ -258,14 +267,18 @@ func (p *ProgressbarPrinter) Add(count int) *ProgressbarPrinter {
 
 // Start the ProgressbarPrinter.
 func (p ProgressbarPrinter) Start(title ...interface{}) (*ProgressbarPrinter, error) {
-	if RawOutput && p.ShowTitle {
+	if RawOutput.Load() && p.ShowTitle {
 		Fprintln(p.Writer, p.Title)
 	}
 	p.IsActive = true
 	if len(title) != 0 {
 		p.Title = Sprint(title...)
 	}
-	ActiveProgressBarPrinters = append(ActiveProgressBarPrinters, &p)
+
+	activeProgressBarPrinters.lock.Lock()
+	activeProgressBarPrinters.printers = append(activeProgressBarPrinters.printers, &p)
```

Comment:
This should is a memory leak and should be fixed eventually. this is appended to but never deleted